### ISSUE TITLE
Using this app

### ISSUE THREAD

**Issue by smvoss** [raxod502/heroku-buildpack-git-lfs#1]
Hey, I love the concept! The only thing I'm wondering about is how to actually deploy a git repo that has LFS to push to heroku?

I am looking to switch into using LFS for this repository, but using the `git push heroku XXX` methodology auto declines all LFS. 

Any tips are appreciated (or what you did to get it there in the first place to actually be able to pull) and I'll make a PR for the readme update. Thanks!

**Comment by raxod502** [comment]
I didn't push directly, I used Heroku's CI service. I just tried doing it manually and got

```
batch response: Repository or object not found: https://git.heroku.com/xxxxxx-xxxxxxx-xxxxx.git/info/lfs/objects/batch                                                                       
Check that it exists and that you have proper access to it
Uploading LFS objects:   0% (0/1), 0 B | 0 B/s, done.
error: failed to push some refs to 'https://git.heroku.com/xxxxxx-xxxxxxx-xxxxx.git'
```

which I assume is your error. Does it work when you `git push --no-verify heroku XXX` as per [this SO question](https://stackoverflow.com/a/49692403/3538165)? If so, I'll update the README to say so.

**Comment by raxod502** [comment]
This thread is being closed automatically by [Tidier](https://github.com/raxod502/tidier) because it is labeled with "waiting on response" and has not seen any activity for 90 days. But don't worry—if you have any information that might advance the discussion, leave a comment and the thread may be reopened :)

**Comment by smvoss** [comment]
Hey @raxod502 - I'm sorry I didn't get back faster it's been a crazy year. I actually did try the `--no-verify` and it worked.

The big piece I still wanted to test was if there were going to be any edge-cases based on which branch was pushed vs where the LFS pieces were at. I'm sorry I wish I remembered more but haven't looked back at it in months.

Long story short - that does seem to work and I'm sure it's worth adding to the readme.
